src/main.rs

```diff
@@ -236,8 +241,8 @@ fn watch(initial_path: Option<&Path>, format: bool) -> io::Result<()> {
                                 #[cfg(feature = "audio")]
                                 "--audio-port",
                                 #[cfg(feature = "audio")]
-                                &audio_port,
-                            ])
+                                &audio_port
+                            ]).args(args.clone())
```

Comment:
Should be `&[args]`, no need to clone.

Reply:
You mean `&args`? But yeah, that makes more sense.

Reply:
Yep